config.ini

```diff
@@ -1,10 +1,15 @@
 [COMMON]
-yapi_dir = 
+yapi_dir =
 platform =
 file_extension = .sh
-language = en
```

Comment:
I would set the default language to english because if they don't set it themselves, then the code will error. This would fix that error. I don't see having english as a default an issue because most of the world uses english, or they would be able to set it themselves.

Reply:
This is also part of the reason travis errors in the build. It wants to know the language, but it is not set. Set it in the config and have travis use that.

Reply:
You are right, I setted to no language only to try the install.py script.